### DISCUSSION TITLE
GitHub Actions Scaler

### DISCUSSION THREAD

**Discussion by suhas-arcadis** [kedacore/keda#3575] [General] [answered]
Hello Team,

Is there any GitHub Actions scaler in pipeline similar to the Azure pipelines? 
Can we implement it using an existing scaler as workaround if possible. Any pointers around this would be helpful...

**Comment by tomkerkhove** [comment, marked as answer]
Yes, you can subscribe to https://github.com/kedacore/keda/issues/1732. However, we are waiting on GitHub to make the process better.
